### PR TITLE
croc: 9.6.2 -> 9.6.3

### PR DESCRIPTION
**Pull request by SuperSandro2000**
Diff: https://github.com/schollz/croc/compare/v9.6.2...v9.6.3

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by equirosa** [comment]
Result of `nixpkgs-review pr 217654` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>croc</li>
  </ul>
</details>

**Review by equirosa — APPROVED** [review]

**Comment by NickCao** [comment]
Tests are timing out on darwin, due to that local networking thing?

**Comment by SuperSandro2000** [comment]
Doesn't help, no idea

File: pkgs/tools/networking/croc/test-local-relay.nix [hunk 1]
```diff
@@ -2,18 +2,24 @@
...
+    # wait for things to settle
+    sleep 1
+    MSG2=$(croc --relay localhost:11111 --yes correct-horse-battery-staple)
+    # compare
+    [ "$MSG" = "$MSG2" ] && touch $out
   '';
+
+  __darwinAllowLocalNetworking = true;
+
+  meta.timeout = 300;
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/croc/test-local-relay.nix:24]:
```suggestion
  meta = {
    timeout = 300;
    broken = stdenv.isDarwin;
  };
```

File: pkgs/tools/networking/croc/test-local-relay.nix [hunk 2]
```diff
@@ -2,18 +2,24 @@
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/croc/test-local-relay.nix:2]: add stdenv here

**Pull request merged**